x-pack/plugins/maps/public/shared/layers/sources/kibana_tilemap_source/kibana_tilemap_source.js

```diff
@@ -7,27 +7,28 @@ import React from 'react';
 import { AbstractTMSSource } from '../tms_source';
 import { TileLayer } from '../../tile_layer';
 import { CreateSourceEditor } from './create_source_editor';
-export class KibanaTilemapSource extends  AbstractTMSSource {
+import { getKibanaTileMap } from '../../../../meta';
+
+export class KibanaTilemapSource extends AbstractTMSSource {
 
   static type = 'KIBANA_TILEMAP';
   static title = 'Custom Tile Map Service';
   static description = 'Map tiles configured in kibana.yml';
   static icon = 'logoKibana';
 
-  static createDescriptor(url) {
+  static createDescriptor() {
```

Comment:
shouldn't store url with descriptor. It's duplicating configuration. A backend configuration-change also wouldn't reflect on the map this way.